Specify install in user mode with pip3 as this is not the default in travis

## Expected Behavior
Just install travis without root

## Actual Behavior

[installation is failing](https://travis-ci.com/skycoin/hardware-wallet/jobs/182941259)
```bash
ip3 install "protobuf==3.6.1" ecdsa
Downloading/unpacking protobuf==3.6.1
  Downloading protobuf-3.6.1-py2.py3-none-any.whl (390kB): 390kB downloaded
Downloading/unpacking ecdsa
  Downloading ecdsa-0.13-py2.py3-none-any.whl (86kB): 86kB downloaded
Requirement already satisfied (use --upgrade to upgrade): setuptools in /usr/lib/python3/dist-packages (from protobuf==3.6.1)
Downloading/unpacking six>=1.9 (from protobuf==3.6.1)
  Downloading six-1.12.0-py2.py3-none-any.whl
Installing collected packages: protobuf, ecdsa, six
Cleaning up...
Exception:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/usr/lib/python3/dist-packages/pip/commands/install.py", line 283, in run
    requirement_set.install(install_options, global_options, root=options.root_path)
  File "/usr/lib/python3/dist-packages/pip/req.py", line 1436, in install
    requirement.install(install_options, global_options, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/pip/req.py", line 672, in install
    self.move_wheel_files(self.source_dir, root=root)
  File "/usr/lib/python3/dist-packages/pip/req.py", line 902, in move_wheel_files
    pycompile=self.pycompile,
  File "/usr/lib/python3/dist-packages/pip/wheel.py", line 206, in move_wheel_files
    clobber(source, lib_dir, True)
  File "/usr/lib/python3/dist-packages/pip/wheel.py", line 193, in clobber
    os.makedirs(destsubdir)
  File "/usr/lib/python3.4/os.py", line 237, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/usr/local/lib/python3.4/dist-packages/google'
Storing debug log for failure in /home/travis/.pip/pip.log
make[1]: *** [install-deps-nanopb] Error 2
make[1]: Leaving directory `/home/travis/build/skycoin/hardware-wallet/tiny-firmware/protob'
make: *** [build-deps] Error 2
The command "make emulator" exited with 2.
```

## Steps to Reproduce the Problem
https://travis-ci.com/skycoin/hardware-wallet/jobs/182941259

## Specifications

  - Platform: Travis
  - Subsystem: Linux

## Possible Solution
use `--user` for pip3 install

--- a/Makefile
+++ b/Makefile
@@ -119,7 +119,7 @@ clean-js:
 
 install-deps-nanopb: install-protoc ## Install tools to generate protobuf classes for C and Python with nanopb
 	make -C $(PROTOC_NANOPBGEN_DIR)/proto/
-	$(PIP) install --user "protobuf==$(PROTOC_VERSION)" ecdsa
+	$(PIP) install "protobuf==$(PROTOC_VERSION)" ecdsa
 
 build-c: install-deps-nanopb $(PROTOB_MSG_C) $(OUT_C)/messages_map.h ## Generate protobuf classes for C with nanopb
 

--- a/Makefile
+++ b/Makefile
@@ -119,7 +119,7 @@ clean-js:
 
 install-deps-nanopb: install-protoc ## Install tools to generate protobuf classes for C and Python with nanopb
 	make -C $(PROTOC_NANOPBGEN_DIR)/proto/
-	pip3 install "protobuf==$(PROTOC_VERSION)" ecdsa
+	$(PIP) install --user "protobuf==$(PROTOC_VERSION)" ecdsa
 
 build-c: install-deps-nanopb $(PROTOB_MSG_C) $(OUT_C)/messages_map.h ## Generate protobuf classes for C with nanopb
 

